### PR TITLE
LDAPUrl now treats urlscheme cases insensitive

### PR DESCRIPTION
**Pull request by tiran**
Fixes: https://github.com/python-ldap/python-ldap/issues/280
Signed-off-by: Christian Heimes <cheimes@redhat.com>
Co-authored-by: David Mulder <dmulder@suse.com>

### CODE REVIEW

**Review by droideck — APPROVED** [review]
Sorry, I've started but I haven't finished the review that day.
The code looks good to me!

The mentioned issue is really minor so feel free to merge it as it is. :)

File: Tests/t_ldapurl.py [hunk 1]
```diff
@@ -57,6 +62,11 @@ def test_isLDAPUrl(self):
                     ldap_url, result, expected,
                 )
             )
+            if expected:
+                LDAPUrl(ldapUrl=ldap_url)
+            else:
+                with self.assertRaises(Exception):
```
[the remark this answers is not in the document]
Comment (encukou) [reply Tests/t_ldapurl.py:68]:
```suggestion
                with self.assertRaises(ValueError):
```

File: Doc/reference/ldapurl.rst [hunk 2]
```diff
@@ -65,6 +65,11 @@ A :py:class:`LDAPUrl` object represents a complete LDAP URL.
 .. autoclass:: ldapurl.LDAPUrl
    :members:
 
+   .. versionchanged:: 3.3.0
```
Comment (encukou) [inline Doc/reference/ldapurl.rst:68]:
```suggestion
   .. versionchanged:: 3.4.0
```

**Comment by encukou** [comment]
Thanks!

**Pull request merged**